packages/lesswrong/lib/betas.ts

```diff
@@ -56,6 +56,7 @@ export const userHasEAHomeRHS = isEAForum ? shippedFeature : disabled;
 export const userHasPopularCommentsSection = isEAForum ? shippedFeature : disabled;
 
 export const visitorGetsDynamicFrontpage = isLW ? shippedFeature : disabled;
+export const userHasRecombeeFrontpage = isLW ? adminOnly : disabled;
```

Comment:
I think we probably do want to internalize the `recombeeEnabledSetting` here, since I don't think it makes sense to separate the `adminOnly` (or other group-based check) from whether the feature is enabled at all, when it comes to various components/etc deciding what to show.

We can do that like so:
```
export const userHasRecombeeFrontpage = isLW ? allOf(adminOnly, () => recombeeEnabledSetting.get()) : disabled;
```

Reply:
And then we can clean up all the places where we're already checking `userHasRecombeeFrontpage` to remove the `recombeeEnabledSetting.get()` check